.github/ISSUE_TEMPLATE/dump_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Global Observation Dump Request
+description: Request additional dates be added to a machine's global dump archive
```

Comment:
```suggestion
description: Request additional dates be added to a machine's global dump archive (GDA) or introduce experimental dump data to the GDA
```

Reply:
This is probably more suited for the other issue template to keep this one clean.

Reply:
I'd prefer it in this template or a separate template for adding experimental data to the GDA, instead of including it with the fix/ICs.